packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -196,12 +197,22 @@ class RenderEditable extends RenderBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// This is used for updating the text in the text field in the event
+  /// of cut or paste.
+  TextSelectionDelegate textSelectionDelegate;
+
```

Comment:
nit: extraneous blank line

Reply:
What is 

> the most set [TextSelectionDelegate]

Should it be "most recently set ..."?